src/templates/activityTemplate/activitySpecsSection/index.tsx

```diff
@@ -0,0 +1,100 @@
+import React from 'react';
+import plus from '../../../images/Plus.svg';
+import clock from '../../../images/Time.svg';
+import star from '../../../images/Star.svg';
+import home from '../../../images/Home.svg';
+import camping from '../../../images/Camping.svg';
+import forest from '../../../images/Forest.svg';
+import sun from '../../../images/Sun.svg';
+import parent from '../../../images/Parent.svg';
+import { ActivityPageData } from '../types';
+
+interface ActivitySpecsProps {
+  data: ActivityPageData;
+}
+
+interface locationIconsTypes {
+  [key: string]: string;
+}
+
+const locationIcons: locationIconsTypes = {
+  kämpällä: camping,
+  kololla: home,
+  metsässä: forest,
+  ulkona: sun,
+};
+
+const ActivitySpecs = ({ data }: ActivitySpecsProps) => (
+  <div className="mt-8">
+    {console.log('RR ActivitySpecs', data)}
+    <div className="flex space-x-4">
+      <div className="space-y-4 w-1/3">
+        <div className="bg-gray-light flex p-4 rounded-xl">
```

Comment:
These recurring css classes can also be setup with a variable, makes it easier to change in the future:
```suggestion
const itemClasses = 'bg-gray-light flex p-4 rounded-xl';

const ActivitySpecs = ({ data }: ActivitySpecsProps) => (
  <div className="mt-8">
    {console.log('RR ActivitySpecs', data)}
    <div className="flex space-x-4">
      <div className="space-y-4 w-1/3">
        <div className={itemClasses}>
```